src/util/isEventEmitter.ts

```diff
@@ -0,0 +1,15 @@
+export type EventEmmitterLike1 = {
```

Comment:
does other codebases using this exported types? can these be encapsulated under `fromEvent`?